lia-demo/src/main/java/com/lithium/community/android/example/LoginActivity.java

```diff
@@ -187,6 +189,40 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    private void blockUiForLogout() {
+        btnLogin.setEnabled(false);
+        progressLogin.setVisibility(View.VISIBLE);
+    }
+
+    private class LogoutCallback1 implements Callback<Void, Throwable> {
```

Comment:
Please remove the `1` in the end of the class name if it does not have any function.